compute/api/Compute.Samples/ListImagesPerPageAsync.cs

```diff
@@ -0,0 +1,55 @@
+﻿/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START compute_images_list_page]
+
+using Google.Cloud.Compute.V1;
+using System;
+using System.Threading.Tasks;
+
+public class ListImagesPerPageAsyncSample
+{
+    public async Task ListImagesPerPageAsync(
+        // TODO(developer): Set your own default values for these parameters or pass different values when calling this method.
+        string projectId = "your-project-id")
+    {
+        // Initialize client that will be used to send requests. This client only needs to be created
+        // once, and can be reused for multiple requests.
+        ImagesClient client = await ImagesClient.CreateAsync();
+
+        // Make the request to list all non-deprecated images in a project.
+        ListImagesRequest request = new ListImagesRequest
+        {
+            Project = projectId,
```

Comment:
`MaxResults = 2,`
Kindly add MaxResults parameter, which tells the number of images to be displayed in each page.
If not added, all images will be displayed in a single page.

Reply:
We used 10 in PHP and Python for the paginated results as this seemed to be a good number for demonstration purposes.

Reply:
Yes 10 seems better, I did list sevaral of the public image sets and couldn't find any set containing more than 200 non-deprecated images. I'll change here to 10, but leave the fully automatic one on 100.

Reply:
Thanks Amanda for the explanation! I understand the rational behind the param setting. Will double check the param values in other languages as well.